core/chains/evm/orm_test.go

```diff
@@ -0,0 +1,69 @@
+package evm_test
+
+import (
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/chains/evm"
+	"github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/smartcontractkit/sqlx"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/guregu/null.v4"
```

Comment:
Roman is probably right, we ought to separate these into:

1. built-in
2. third party
3. local repo